kolibri/plugins/learn/assets/src/views/content-cards/index.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+
+  <component :header="header" :subheader="subheader" :contents="contentsWithLinks" :is="container" />
```

Comment:
That was something I was planning on doing. I consolidated them initially because there was a grid, expandable grid, and topic list. Seeing as they were all representations of a group of content, I thought this would make sense. I'm seeing now that its not as helpful as I thought it'd be.

One nice thing is how `content-card` isn't exposed outside of it's parent. But yeah, I agree they should be broken up.